### PR TITLE
Improve coverage

### PR DESCRIPTION
**Pull request by ildyria**
- removing dead code, and non-attainable lines...

### CODE REVIEW

File: codecov.yml [hunk 1]
```diff
@@ -1,6 +1,9 @@
 ---
 codecov:
   require_ci_to_pass: true
+  notify:
+    after_n_builds: 9
```
Comment (d7415) [inline codecov.yml:5]: Is this to stop the "This build decreases coverage by 0.000000000000007%" messages?

**Review by d7415 — DISMISSED** [review]
Doesn't look like it was used (and you should know)

File: codecov.yml (hunk 1 above)
Comment (ildyria) [reply codecov.yml:5]: YES!

**Comment by ildyria** [comment]
Mostly adding some code ignore in un-testable corners + fix codecov warnings a bit too early...

**Review by d7415 — APPROVED** [review]
I suppose the desired approach is that we create tests that throw these errors, but that's not going to be straightforward for all of them.

**Comment by ildyria** [comment]
> I suppose the desired approach is that we create tests that throw these errors, but that's not going to be straightforward for all of them.

good luck with that, especially the ones where you need to not have posix not enabled... Those somewhat requires to have their own ubuntu stack etc. :(

**Pull request merged**